onnxruntime/test/onnx/main.cc

```diff
@@ -450,8 +450,6 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"bitshift_right_uint16", "BitShift(11) uint16 support not enabled currently"},
       {"bitshift_left_uint8", "BitShift(11) uint8 support not enabled currently"},
       {"bitshift_left_uint16", "BitShift(11) uint16 support not enabled currently"},
-      {"reflect_pad", "test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests"},
```

Comment:
It would be great if you can check-in simple unit tests to cover all the new types - int32, int64, and double.

Reply:
I'm going to add the tests